### PR TITLE
feat: add sockets and message schema

### LINKED ISSUE #12: [FEATURE] <socket.io server setup in real time chat>
### ⭐ Suggestion

For making any real-time chat app, Socket.io is preferable as it enables low-latency, bidirectional and event-based communication between a client and a server.

### 💻 Use Cases

_No response_

### ❌ Related Problems

_No response_

### PR DESCRIPTION
**Pull request by bhavyastar**
## Changes proposed

1.  Socket.io setup done for generation of established connection (cords) between sender and receiver 
2. Message Schema added for the sender and receiver. 

## Check List (Check all the boxes which are applicable) 

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Thank you for your valuable time in reviewing my code. 😄

### CODE REVIEW

**Review by chandankumar1307 — APPROVED** [review]
LGTM

File: src/chat/chat.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        <div class="chat-logo">
+             <img src="" alt="" />
+        <h1> Cords </h1>
+        </div>
+     </div>
+    </>
+  )
+}
+
+export default chat
```
Comment (SuperAayush) [inline src/chat/chat.js:16]: It would be better to add comments for more user friendly experience.

**Review by SuperAayush — CHANGES_REQUESTED** [review]

File: src/chat/chat.js (hunk 1 above)
Comment (SuperAayush) [reply src/chat/chat.js:16]: Also, it would be better to remove empty files and then go for PR.

File: server/Routes/Routes.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import express from "express";
 import Post from "../Schema/test.js";
 import User from "../Schema/userSchema.js";
 import { createFeeds, getFeeds } from "../Controllers/feeds.js";
+const { addMessage, getMessages } = require("../controllers/messageController");
 import {
```
Comment (AayushShuk1a) [inline server/Routes/Routes.js:6]: Better to use import than require.

**Review by SuperAayush — APPROVED** [review]
LGTM

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by AayushShuk1a — APPROVED** [review]
LGTM👍

**Review by chirag0002 — APPROVED** [review]
LGTM

**Pull request merged**